manifests/charts/istio-control/istio-discovery/files/gen-istio.yaml

```diff
@@ -1538,6 +1538,370 @@ spec:
                     local:
                       inline_string: "envoy.wasm.stats"
 ---
+# Source: istio-discovery/templates/telemetryv2_1.8.yaml
+# Note: metadata exchange filter is wasm enabled only in sidecars.
```

Comment:
where did this come from?

Reply:
From the `make gen` step, not sure why it gets added